app/views/guidance_groups/_index_by_theme.html.erb

```diff
@@ -9,34 +9,47 @@
     </div>
   </div>
   <% guidance_groups_by_theme.each_pair do |guidance_group, theme_hash| %>
+    <% guidances_output = [] %>
     <% theme_hash.each_pair do |theme, guidances| %>
-      <div class="panel panel-default">
-        <div
-          class="heading-button"
-          role="button"
-          data-toggle="collapse"
-          href="<%= "##{guidances.object_id}" %>"
-          aria-expanded="false"
-          aria-controls="<%= "##{guidances.object_id}" %>">
-          <div class="panel-heading" role="tab" id="<%= "panel-heading-#{guidances.object_id}" %>">
-            <h2 class="panel-title">
-              <%= theme.title %>
-              <i class="fa fa-plus pull-right" aria-hidden="true"></i>
-            </h2>
+      <% if (guidances.map{|g| g.id} - guidances_output).any? # any not output%>
```

Comment:
- Can you move the comment to above the line please? (Trying to stay strict to the style guide)

- Suggested minor refactor: `guidances.map(&:id)`

Reply:
addded the comment above and updated the statement.  
I need to get in the habit of using the `&:`